validphys2/src/validphys/convolution.py

```diff
@@ -91,6 +134,8 @@ def _id(a):
     "ASY": _asy,
     "ADD": operator.add,
     "SMN": _smn,
+    "SMP": _smp,
+    "ADDP": _addp,
```

Comment:
I hadn't looked at the data yet, but please use some other name.
In particular `ADDP` is a very weird name for `b - abs(a)`

Reply:
Do you have some suggestions? I basically tried to follow what's there although admittedly inconsistently `SMN->SMP` and `ADD->ADDP` where `P` always stands for Polarized.

Reply:
`SUBTRACT_ABS` would be more correct that `ADDP` imho ^^U
the operation doesn't need to know anything about the polarized situation

Reply:
Ok, do you like more the new names?
https://github.com/NNPDF/nnpdf/blob/043e974b803b74bcd4058eaf3e7e93015648df0d/validphys2/src/validphys/convolution.py#L140-L141

I've also re-ordered completely the FK tables here (#1816) such that we have `a - abs(b)` and `a + b - abs(c + d)` as the previous ones were odds as well.

Also, the arguments types are fixed. 

The other comments I will address in one push (with different commits ofc).